sdk/identity/identity/tsconfig.json

```diff
@@ -6,9 +6,9 @@
     "declarationDir": "./types",
     "outDir": "./dist-esm",
     "resolveJsonModule": true,
-    "noUnusedLocals": false,
-    "noUnusedParameters": false,
-    "noImplicitReturns": false
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true
```

Comment:
Since the base tsconfig file have the above set to true, we can remove these lines entirely

```suggestion
```